src/math/transform/Quaternion.cpp

```diff
@@ -2,24 +2,56 @@
 
 #include <glm/ext/quaternion_trigonometric.hpp>
 
+#include "glm/geometric.hpp"
```

Comment:
fix include ordering and formatting (glm should have angle brackets and be grouped with itself)